client/src/components/SelfReviewForm/SelfReviewForm.js

```diff
@@ -0,0 +1,320 @@
+                    Employee *
+                  </Grid>
+                  <Grid xs={6} sm={6} md={3} item>
+                    <SelectMenu
+                      name="employee"
+                      onChange={handleChange}
+                      disabledName="Select Employee"
+                      value={values.employee}
+                    >
+                      {employeeData
+                        ? employeeData.map((prop, key) => {
+                          return prop.status !== 'Inactive' ? (
+                            <MenuItem
+                              className={classes.hoverEffect}
+                              value={prop._id}
+                              key={key}
+                            >
+                              {prop.firstname} {prop.lastname}
+                            </MenuItem>
+                          ) : null
+                        })
+                        : null}
+                    </SelectMenu>
+                  </Grid>
+                  <Grid xs={6} sm={6} md={3} className={classes.grid} item>
+                    Projects *
+                  </Grid>
+                  <Grid xs={6} sm={6} md={3} item>
+                    <SelectMenu
+                      name="projects"
+                      onChange={handleChange}
+                      disabledName="Select Project"
+                      value={values.projects}
+                      multiple
+                    >
+                      {projectsData
+                        ? projectsData.map((prop, key) => {
+                          return (
+                            <MenuItem
+                              className={classes.hoverEffect}
+                              value={prop._id}
+                              key={key}
+                            >
+                              {prop.title}
+                            </MenuItem>
+                          )
+                        })
+                        : null}
+                    </SelectMenu>
+                  </Grid>
+                </Grid>
+                <Grid className={classes.container} container>
+                  <Grid xs={6} sm={6} md={3} className={classes.grid} item>
+                    Functional Manager *
+                  </Grid>
+                  <Grid xs={6} sm={6} md={3} item>
+                    <SelectMenu
+                      name="functional_manager"
+                      onChange={handleChange}
+                      disabledName="Select Manager"
+                      value={values.functional_manager}
+                    >
+                      {managers
+                        ? managers.map(item => {
+                          return (
+                            <MenuItem
+                              value={item._id}
+                              className={classes.hoverEffect}
+                            >
+                              {item.firstname + ' ' + item.lastname}
```

Comment:
I think we need to use template literal here. Is it possible to use template literal?

Reply:
done